src/Robo/Plugin/Traits/SitesConfigTrait.php

```diff
@@ -78,12 +78,17 @@ protected function getSiteConfig(string $siteName = 'default'): array
      *   The site configuration key to load.
      * @param string $siteName
      *   The site name.
+     * @param bool $required
+     *   Whether the config item is expected to always be present.
      */
-    public function getSiteConfigItem(string $key, string $siteName = 'default'): mixed
+    public function getSiteConfigItem(string $key, string $siteName = 'default', bool $required = true): mixed
```

Comment:
Any concern with this breaking existing functionality? Maybe that's a feature not a bug though?